adr/2021-11-17-copy-certs.md

```diff
@@ -0,0 +1,3 @@
+Maurice:
```

Comment:
I was not aware that this was an issue and that it needed solving, can you maybe connect it to an issue?

Reply:
No, it was just "supposed" to work but never did. No overhead of an issue needed.

Reply:
I thought we agreed to put priorities together as a team, then select what needs most work on. I have nothing against you picking your own agenda, but then let the rest know that you don't want to be part of it.

Reply:
Sure, but this solves a rather pressing issue that was reported in Jira by one of our customers. No rocket science involved, just progression.